### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by astephenb**
Just noticed the missing 'c' when reading through the docs!

### CODE REVIEW

**Comment by bikeath1337** [comment]
Thanks @astephenb . Will merge this later. I did my best to find typos. :) LOL

**Comment by bikeath1337** [comment]
How's V2 working for you?

**Comment by astephenb** [comment]
It happens! Thanks for the great package!

**Comment by astephenb** [comment]
We are actually not on the latest yet. We are using 1.5.8.

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ New Features in v2.0:
 - The `progress()` and `complete()` callbacks now provide upload stats like transfer rate and time remaining.
 - Reduced memory footprint when calculating MD5 digests.
 
-To migrate to v2.0, [follow these instructions](https://github.com/bikeath1337/EvaporateJS/tree/promises#migrating-from-v1x-to-v20).
+To migrate to v2.0, [follow these instructions](#migrating-from-v1x-to-v20).
```
Comment (astephenb) [inline README.md]: Just noticed this as well.

**Comment by bikeath1337** [comment]
If you are on 1.5.8, you should at least be current on the 1.6.x line as everything is fully backward compatible and more stable.

**Pull request merged**